### ISSUE TITLE
Support filtering with JSONPath

### ISSUE THREAD

**Issue by bbbco** [tulios/json-viewer#140]
It would be great if we could filter the JSON document using JSONPath to only show relevant parts that we want to see.

http://goessner.net/articles/JsonPath/

https://github.com/dchester/jsonpath

**Comment by lipsumar** [comment]
Yes! Or manipulate via [jq](https://stedolan.github.io/jq/), that would be amazing.

**Comment by npejo** [comment]
We are currently working on this at the moment. It will be available soon 😀

**Comment by vanduc1102** [comment]
it's been 2 years, any update so far

**Comment by bbbco** [comment]
Use bro/q
https://github.com/zalando-incubator/bro-q

**Comment by vanduc1102** [comment]
installed, removed json-view
